src/c/src/utf16.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2017 The Procyon Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef PROCYON_UTF16_H_
+#define PROCYON_UTF16_H_
+
+#include <pn/procyon.h>
+#include <sys/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+void pn_u16chr(pn_rune_t rune, uint16_t* data, size_t* count);
```

Comment:
Seems a little silly to have two single-function headers `utf8.h` and `utf16.h`. Maybe `unicode.h` / `unicode.c` with both UTF-8 and UTF-16?